content/en/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -0,0 +1,205 @@
+
+<table>
+	<caption style="display:none">Privileged policy specification</caption>
+	<tbody>
+		<tr>
+			<td><strong>Control</strong></td>
+			<td><strong>Policy</strong></td>
+		</tr>
+		<tr>
+			<td style="white-space: nowrap"><a href="/docs/concepts/security/pod-security-standards">Windows HostProcess</a></td>
+			<td>
+				<p>Windows pods offer the ability to run <a href="/docs/tasks/configure-pod-container/create-hostprocess-pod">
+        HostProcess containers</a> which enables privileged access to the Windows node. </p>
+				<p><strong>Allowed Values</strong></p>
+				<ul>
+					<li><code>true</code></li>
+				</ul>
+			</td>
+		</tr>
+		<tr>
+			<td style="white-space: nowrap"><a href="/docs/concepts/security/pod-security-standards">Host Networking</a></td>
+			<td>
+				<p>Will be in host network by default initially. Support 
+				to set network to a different compartment may be desirable in 
+				the future.</p>
+				<p><strong>Allowed Values</strong></p>
+				<ul>
+					<li><code>true</code></li>
+				</ul>
+			</td>
+		</tr>
+    <tr>
+			<td style="white-space: nowrap"><a href="/docs/tasks/configure-pod-container/configure-runasusername/">runAsUsername</a></td>
+			<td>
+				<p>Specification of which user the HostProcess container should run as is required for the pod spec.</p>
+				<p><strong>Allowed Values</strong></p>
+				<ul>
+          <li><code>"NT AUTHORITY\\SYSTEM"</code></li>
+					<li><code>"NT AUTHORITY\\Local service"</code></li>
+          <li><code>"NT AUTHORITY\\NetworkService"</code></li>
+				</ul>
+			</td>
+		</tr>
+    <tr>
+			<td style="white-space: nowrap"><a href="/docs/concepts/security/pod-security-standards">runAsNonRoot</a></td>
+			<td>
+				<p>HostProcess containers have privileged access to the host so the runAsNonRoot field cannot be set to true.</p>
+				<p><strong>Allowed Values</strong></p>
+				<ul>
+          <li>Undefined/Nil</li>
+					<li><code>false</code></li>
+				</ul>
+			</td>
+		</tr>
+	</tbody>
+</table>
+
+### Example Manifest
+
+```yaml
+spec:
+  securityContext:
+    windowsOptions:
+      hostProcess: true
+      runAsUserName: "NT AUTHORITY\\Local service"
+  hostNetwork: true
+  containers:
+  - name: test
+    image: image1:latest
+    command:
+      - ping
+      - -t
+      - 127.0.0.1
+  nodeSelector:
+    "kubernetes.io/os": windows
+```
+
+## Volume Mounts
+
+HostProcess containers support the ability to mount volumes within the container volume space. 
+Applications running inside the container can access volume mounts directly via relative or 
+absolute paths. An environment variable `$CONTAINER_SANDBOX_MOUNT_POINT` is set upon container 
+creation and provides the absolute host path to the container volume. Relative paths are based 
+upon the `Pod.containers.volumeMounts.mountPath` configuration.
+
+Example: To access service account tokens the following path structures are supported within the container:
+
+`.\var\run\secrets\kubernetes.io\serviceaccount\`
```

Comment:
Could you use a UNC path?

Reply:
In this case the security account token is mapped in by kubelet so won't be on a UNC path.  

In general the system accounts should be able to use UNC paths if they have access. Volume mounting them into the container might not make sense and I think the guidance would be the same as the guidance for named pipes.  @dcantah  to confirm

Reply:
A relative path like this would only work if the working directory was untouched. The default working directory is where the volume is mounted, but a different one could be set in the image or through cri etc. so this wouldn't work in that case. Good to call out.

Reply:
We had a little race on the comment :P, but yea the guidance would be to just use the path on the host. As Kubelet maps this in explicitly I think that's why we call this scenario out